docs/docs/building-modules/14-depinject.md

```diff
@@ -0,0 +1,120 @@
+---
+sidebar_position: 1
+---
+
+# Dependency Injection
+
+:::note
+
+### Pre-requisite Readings
+
+* [Cosmos SDK Dependency Injection Framework](../tooling/02-depinject.md)
+
+:::
+
+[`depinject`](../tooling/02-depinject.md) is used to wire any module in `app.go`.
+All core modules are already configured to support dependency injection.
+
+Modules, to work with `depinject`, must defines its configuration and its requirements so that `depinject` can provide the right dependencies.
```

Comment:
```suggestion
To work with `depinject` a module must define its configuration and requirements so that `depinject` can provide the right dependencies.
```